src/components/Heading/Heading.tsx

```diff
@@ -35,6 +39,10 @@ export default function Heading(props: React.PropsWithChildren<HeadingProps>) {
 
   const baseClass = "heading";
 
+  if (displaySize) {
```

Comment:
There should also be a check to make sure that `displaySize` is one of the appropriate `DisplaySizes` options. Right now, you can add any string and the component won't complain about it (and it also doesn't affect the size).

Reply:
I will create a ticket to add this functionality in the future.